### PR TITLE
Crm457 224/js testing

### PR DESCRIPTION
**Pull request by dwhenry**
## Description of change
System tests with javascript

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRM457-224

## Notes for reviewer
I decided to go with cuprite over selenium as it appears to be the new thing. 
And it in the rails docs on system testing

It work pretty well, except for one instance where it would find an element on the page (thinks it is invisible)

### CODE REVIEW

File: db/schema.rb
Comment (pking-moj) [inline db/schema.rb:1]: Is this work from another PR that has crept in?

File: spec/system/support/better_rails_system_tests.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module BetterRailsSystemTests
```
[the remark this answers is not in the document]
Comment (dwhenry) [reply spec/system/support/better_rails_system_tests.rb:3]: This was copied and pasted... I should link to the page... let me think about this...

**Review by ivanELEC — COMMENTED** [review]
same comment as Paul but otherwise approved

**Comment by ivanELEC** [comment]
Also branch has conflicts

**Review by pking-moj — APPROVED** [review]

**Pull request merged**